js/apps/kc-create/create.js

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env node
+
+import chalk from "chalk";
+import { Command } from "commander";
+import fs from "fs-extra";
+import Mustache from "mustache";
+import { mkdtemp } from "node:fs";
+import { tmpdir } from "node:os";
+import { join } from "node:path";
+import path from "path";
+import { simpleGit } from "simple-git";
+import { fileURLToPath } from "url";
+
+const packageJson = JSON.parse(fs.readFileSync("./package.json", "utf8"));
+
+function main() {
+  new Command(packageJson.name)
+    .version(packageJson.version)
+    .description(packageJson.description)
+    .arguments("<name>")
+    .usage(`${chalk.green("<name>")} [options]`)
+    .option(
+      "-t, --type <name>",
+      "the type of ui to be created either account or admin ",
+      "account",
+    )
+    .action(async (name, options) => {
+      console.log(`Creating a new ${chalk.green(name)} project`);
+
+      await createProject(name, options.type);
+      done(name);
+    })
+    .on("--help", () => {
+      console.log();
+      console.log(`    Only ${chalk.green("<name>")} is required.`);
+      console.log();
+      console.log(`    Type ${chalk.blue("--type")} can be one of:`);
+      console.log(
+        `      - ${chalk.green("account")} for an account ui based ui`,
+      );
+      console.log(`      - ${chalk.green("admin")} for an admin ui based ui`);
+      console.log();
+    })
+    .parse(process.argv);
+}
+
+function cloneQuickstart() {
+  return new Promise((resolve, reject) => {
+    mkdtemp(join(tmpdir(), "template-"), async (err, dir) => {
+      if (err) return reject(err);
+      simpleGit()
+        .clone("https://github.com/keycloak/keycloak-quickstarts", dir, {
+          "--single-branch": undefined,
```

Comment:
Any particular reason this is needed? I imagine that not passing anything here would result in the same logic.